web/assets/service-worker.js

```diff
@@ -15,6 +16,14 @@ const PREFETCH_CACHE_FILES = [
     "/static/node_modules/@alpinejs/persist/dist/cdn.min.js",
     "/static/node_modules/alpinejs/dist/cdn.js",
 ];
+const CACHE_REQUEST_METHOD_WHITELIST = ["GET"];
+const CACHE_REQUEST_HOST_WHITELIST = [self.location.host];
```

Comment:
Are you fine with `ALLOWLIST` instead? @mono424

Reply:
Goland now shows ALLOWLIST as a typo, which is annoying. I still changed it for now, but if you have other proposals for the name, feel free to change it 👍